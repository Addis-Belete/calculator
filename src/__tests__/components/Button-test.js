describe('Button', () => {
  it('passing test', () => {
    expect(true).toBeTruthy();
  });
});
