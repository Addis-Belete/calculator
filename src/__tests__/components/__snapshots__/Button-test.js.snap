// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should renders Button 1`] = `
<button
  name="1"
  onClick={[Function]}
  type="button"
>
  1
</button>
`;
